app/models/parent_object.rb

```diff
@@ -23,6 +24,17 @@ def default_fetch
     end
   end
 
+  def authoritative_json
```

Comment:
It seems like you're writing a `.to_json` method - is there a reason not to just override the default `.to_json` method here?  (i.e. is there every a reason you'd want the output from just `parent_object.to_json` instead of `parent_object.authoritative_json` - if not, I'd be inclined to just override the default `.to_json` method.

Reply:
Also, this will silently fail (i.e. just return nil) if the metadata_cloud_name is not in the expected list.  You should probably raise an error if none of the cases match
```
when 
   ... 
else
    raise "Unexpected metadata cloud name: #{authoritative_metadata_source.metadata_cloud_name}
end

Reply:
I agree about the raise, that's a good catch. We're not overriding to_json since this doesn't write the active record fields to json, but instead selects which of the jsonb fields is the one that should be used to write out to the file. In my opinion to_json is about the object as json, this is about one fields inside the object that happens to be json.

Reply:
I'm probably conflating this with the `.to_solr` pattern for old Hydra-based apps - i.e. give me the thing serialized in a way that I can index it to Solr.